_docs/setup/install-kubernetes.md

```diff
@@ -231,52 +199,34 @@ kubectl create -f <(istioctl kube-inject -f <your-app-spec>.yaml)
 
 ## Uninstalling
 
-You can use the [Istio Helm chart](https://github.com/kubernetes/charts/tree/master/incubator/istio#uninstalling-the-chart) to uninstall, or follow the steps below.
-
-1. Uninstall Istio core components:
-
-   * If Istio was installed without Istio auth feature:
+Upgrade is not yet supported. Please un-install using the same yaml files you used during install.
```

Comment:
uninstall (one word).
And the upgrade sentence seems out of context. 
I would move this to the top of the page, before starting install. Something like
"If you are already running Istio 0.1, please uninstall Istio first."